src/parlant/core/engines/alpha/prompt_builder.py

```diff
@@ -238,3 +240,24 @@ def add_staged_events(
             )
 
         return self
+
+    def add_few_shots(
+        self,
+        shots: Sequence[Shot],
+    ) -> PromptBuilder:
+        if shots:
+            self.add_section(
+                name=BuiltInSection.FEW_SHOTS,
+                content="".join(
+                    f"""
+Example #{i}:
```

Comment:
Aren't you missing the separators? Those can be important (e.g. ------------ or ###)